caesars-palace/blocks/slider-card/slider-card.js

```diff
@@ -1,42 +1,75 @@
 export default function decorate(block) {
   const cardWrapper = document.createElement('div');
   cardWrapper.classList.add('card-wrapper');
-  block.querySelectorAll('div.slider-card > div').forEach((div) => {
+
+  function isADesktop() {
```

Comment:
Could use arrow function here. It is a preference but I prefer to not nest `function` notation.
```suggestion
  const isADesktop = () => {...}
```

Reply:
You could move this function outside of the default function which would actually be the better solution.